x/mint/simulation/genesis.go

```diff
@@ -5,46 +5,199 @@ package simulation
 import (
 	"encoding/json"
 	"fmt"
+	"math/rand"
 
 	"github.com/osmosis-labs/osmosis/v7/x/mint/types"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
 )
 
+// Simulation parameter constants.
+const (
+	epochProvisionsKey         = "genesis_epoch_provisions"
+	epochIdentifierKey         = "epoch_identifier"
+	reductionFactorKey         = "reduction_factor"
+	reductionPeriodInEpochsKey = "reduction_period_in_epochs"
+
+	distributionProportionsKey = "distribution_proportions"
+
+	stakingDistributionProportionKey          = "staking_distribution_proportion"
+	poolIncentivesDistributionProportionKey   = "pool_incentives_distribution_proportion"
+	developerRewardsDistributionProportionKey = "developer_rewards_distribution_proportion"
+	communityPoolDistributionProportionKey    = "community_pool_distribution_proportion"
+	weightedDevRewardReceiversKey             = "weighted_developer_rewards_receivers"
+	mintingRewardsDistributionStartEpochKey   = "minting_rewards_distribution_start_epoch"
+
+	maxInt64 = int(^uint(0) >> 1)
+)
+
+var (
+	epochIdentifierOptions    = []string{"day", "week"}
+	possibleBech32AddrLengths = []uint8{20, 32}
+)
+
 // RandomizedGenState generates a random GenesisState for mint.
 func RandomizedGenState(simState *module.SimulationState) {
-	// minter
+	var epochProvisions sdk.Dec
+	simState.AppParams.GetOrGenerate(
+		simState.Cdc, epochProvisionsKey, &epochProvisions, simState.Rand,
+		func(r *rand.Rand) { epochProvisions = genEpochProvisions(r) },
+	)
+
+	var epochIdentifier string
+	simState.AppParams.GetOrGenerate(
+		simState.Cdc, epochIdentifierKey, &epochIdentifier, simState.Rand,
+		func(r *rand.Rand) { epochIdentifier = genEpochIdentifier(r) },
+	)
```

Comment:
Lets hardcode the epoch Identifier, its easier to make sure everything wires correctly